src/scripts/main.js

```diff
@@ -1 +1,65 @@
 'use strict';
+
+const leftClick = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const rightClick = new Promise((resolve, reject) => {
+  document.addEventListener('contextmenu', () => {
+    resolve();
+  });
+});
+
+const firstPromise = new Promise((resolve, reject) => {
+  leftClick
+    .then(() => {
+      resolve('First promise was resolved');
+    });
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+});
+
+const secondPromise = new Promise((resolve, reject) => {
+  leftClick
+    .then(() => {
+      resolve('Second promise was resolved');
+    });
+
+  rightClick
+    .then(() => {
+      resolve('Second promise was resolved');
```

Comment:
Use helper function for code improvement. `promise was resolved` repeated several times